test/navigation/NavigatorSpec.scala

```diff
@@ -261,29 +261,6 @@ class NavigatorSpec extends SpecBase with ViewBehaviours {
           .mustBe(routes.RepresentativeDeclarantAndBusinessNameController.onPageLoad(NormalMode))
       }
 
-      "go to AmendCaseResponseType page after ReferenceNumber page " in {
-        navigator.nextPage(ReferenceNumberPage, NormalMode, emptyUserAnswers)
-          .mustBe(routes.AmendCaseResponseTypeController.onPageLoad(NormalMode))
-      }
-
-      "go to FurtherInformation page after ReferenceType page as Further Information " in {
-        val values: Set[AmendCaseResponseType] = Set(AmendCaseResponseType.FurtherInformation)
-        val answers =
-          emptyUserAnswers
-            .set(AmendCaseResponseTypePage, values).success.value
-        navigator.nextPage(AmendCaseResponseTypePage, NormalMode, answers)
-          .mustBe(routes.FurtherInformationController.onPageLoad(NormalMode))
-      }
-
-      "go to SupportingDocuments page after ReferenceType page as Supporting documents " in {
-        val values: Set[AmendCaseResponseType] = Set(AmendCaseResponseType.SupportingDocuments)
-        val answers =
-          emptyUserAnswers
-            .set(AmendCaseResponseTypePage, values).success.value
-        navigator.nextPage(AmendCaseResponseTypePage, NormalMode, answers)
-          .mustBe(routes.AmendCaseSendInformationController.showFileUpload(NormalMode))
-      }
-
       "go to AmendCheckYourAnswers page after FurtherInformation page " in {
```

Comment:
CLARIFICATION: Should these two AmendCheckYourAnswersController and AmendConfirmationController be removed from the NavigatorSpec as they are now in the AmendNavigatorSpec?

Reply:
Good spot.  I'll "break" the test first as the old navigator should no longer work for this page.  Then delete the test.

Reply:
Removed unused code from old navigator